docs/frontend_utilisation_guide.md

```diff
@@ -24,48 +24,92 @@ The Pixel Streaming Frontend UI contains all the functionality for UI components
 
 ---
 
-## Getting Started
-### Installing the Scalable Pixel Streaming Frontend   
+## Scalable Pixel Streaming Frontend packages
+### The Scalable Pixel Streaming Frontend library
+The library is the part of the Scalable Pixel Streaming Frontend that consumes the Epic Games Pixel Streaming Frontend and UI Frontend. It includes all of the custom signalling server logic that Scalable Pixel Streaming signalling servers require to work. The library can be either obtained through [Github](https://github.com/ScalablePixelStreaming/Frontend) or [NPM](https://www.npmjs.com/package/@tensorworks/libspsfrontend). To make use of the library it must be initialised via HTML and Javascript. The library is written in TypeScript but configured to export as a UMD module and can be consumed by plain JavaScript and most JavaScript frameworks.  
+
+### The Scalable Pixel Streaming Frontend TypesScript example
```

Comment:
fix typo in TypeScript

Reply:
Resolved in 4820831c542f48909f428f142606b7c814b6ede6